server/database/seed.js

```diff
@@ -8,9 +8,9 @@ const file = path.join(__dirname, 'sampleData5.csv');
 const writeReviews = fs.createWriteStream(file, {flags: 'a'});
 
 const seed = (storeStart, storeEnd) => {
-  for (let s = storeStart; s < storeEnd; s++) {                  // 10000 stores
-    for (let d = 0; d < 100; d++) {                              // 100 items
-      for (let c = 0; c < Math.ceil(Math.random() * 11); c++) {  // 5 reviews per item === 5M Rev per Round
+  for (let s = storeStart; s < storeEnd; s++) {
+    for (let d = 0; d < 100; d++) {
+      for (let c = 0; c < Math.ceil(Math.random() * 11); c++) {
         let itemId = d;
         let date = new Date(faker.date.past(6)).toUTCString().split(',').slice(1);
         let mockData = `${faker.internet.userName()},${fake.names[Math.floor(Math.random() * 650)]},${faker.image.avatar()},${date},${Math.ceil(Math.random() * 5)},${faker.lorem.sentences()},${itemId},${fake.productNames[itemId]},${faker.image.imageUrl()},${s},${faker.image.imageUrl()},${Math.floor(Math.random() * 20)}\n`;
```

Comment:
This line should be cleaned up a bit

Reply:
Ended up doing just that, it wasn't working how I expected and at a later thought made sense why. My next commit will reflect that.